app/models/comment.rb

```diff
@@ -1,3 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :post_id
+  belongs_to :users, class_name: 'User'
```

Comment:
- [ ] Same here, pleaase make the reference through author by adding belongs to `author` and classname `User`